spack-repo/packages/singularity-eos/package.py

```diff
@@ -112,6 +112,7 @@ class SingularityEos(CMakePackage, CudaPackage):
 
     depends_on("ports-of-call@1.4.2,1.5.2:", when="@:1.7.0")
     depends_on("ports-of-call@1.5.2:", when="@1.7.1:")
+    depends_on("ports-of-call@1.6.0:", when="@1.8.1:")
```

Comment:
@rbberger do you have a preference on a different way to do this? I.e. how do I identify a new dependency in a version that hasn't been released yet?

Or should we handle these at the release and just set them to `when="@main"` for now?

Reply:
@cmauney @Yurlungur @dholladay00 do you have preferences here?

Reply:
@rbberger any thoughts?